### ISSUE TITLE
Deploy Application to Cloud Foundry with SAP Cloud SDK for JavaScript

### ISSUE THREAD

**Issue by qmacro** [SAPDocuments/Tutorials#3671]
Tutorial URL: https://developers.sap.com/tutorials/s4sdkjs-deploy-application-cloud-foundry.html

Please specify the step you are referring to: 1 (Deploy)

When running the `ci-build` script (`npm run ci-build`) which invokes the TS compiler, the following error occurs:

```
~/local/projects/cloudsdktut [scaffolding-ts {scaffolding/scaffolding-ts}|✚ 3…3]
12:28 $ npm run ci-build

> myapp@1.0.0 ci-build /Users/i347491/local/projects/cloudsdktut
> npx tsc

node_modules/@sap/cloud-sdk-core/dist/request-builder/http-agent.d.ts:8:33 - error TS7016: Could not find a declaration file for module 'http-proxy-agent'. '/Users/i347491/local/projects/cloudsdktut/node_modules/http-proxy-agent/index.js' implicitly has an 'any' type.
  Try `npm install @types/http-proxy-agent` if it exists or add a new declaration (.d.ts) file containing `declare module 'http-proxy-agent';`

8 import HttpProxyAgent = require('http-proxy-agent');
                                  ~~~~~~~~~~~~~~~~~~


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! myapp@1.0.0 ci-build: `npx tsc`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the myapp@1.0.0 ci-build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/i347491/.npm/_logs/2019-07-02T11_28_57_815Z-debug.log
~/local/projects/cloudsdktut [scaffolding-ts {scaffolding/scaffolding-ts}|✚ 3…3]
12:28 $
```

Installing the `@types/http-proxy-agent` package (as the message suggests) solves this issue - the compilation completes silently and successfully after this. 

Is it worth putting in a warning about this in the tutorial, or fixing the root cause if possible?

Thanks.

**Comment by mr-flannery** [comment]
Hey DJ,

this issue should have been fixed in version `1.5.0`, which is available since last Thursday. Could you retry it with the new version and see if the issue persists (make sure to remove the `@types/http-proxy-agent`, this should now be correctly installed as dependency of the SDK)?

**Comment by thecodester** [comment]
Thank you for your feedback. The issue seems to be resolved, so I am closing the issue. 

If you still have questions, feel free to reopen the issue.
